File: platforms/kotlin/examples/core/pass/set_clear_color.kt

import org.fragmentcolor.*

val renderer = Renderer()
val target = renderer.createTextureTarget(64u, 64u)

val shader = Shader.default()
val pass = Pass("solid background")
pass.addShader(shader)

pass.setClearColor(listOf(0.1f, 0.2f, 0.3f, 1.0f))

renderer.render(pass, target)

File: platforms/swift/examples/core/pass/set_clear_color.swift

import FragmentColor

let renderer = Renderer()
let target = try await renderer.createTextureTarget([64, 64])

let shader = Shader.default()
let pass = Pass("solid background")
pass.addShader(shader)

try pass.setClearColor([0.1, 0.2, 0.3, 1.0])

try renderer.render(pass, target)